pkgs/atuinsh/atuin/registry.yaml

```diff
@@ -10,16 +10,21 @@ packages:
     replacements:
       amd64: x86_64
       darwin: apple-darwin
-      linux: unknown-linux-gnu
+      linux: unknown-linux-musl
     files:
       - name: atuin
         src: atuin-{{.Version}}-{{.Arch}}-{{.OS}}/atuin
     supported_envs:
       - linux/amd64
       - darwin
     rosetta2: true
-    version_constraint: semver(">= 0.6.4")
+    version_constraint: semver(">= 14.0.0")
     version_overrides:
+      - version_constraint: semver("< 14.0.0")
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+          linux: unknown-linux-gnu
       - version_constraint: semver("< 0.6.4")
```

Comment:
`semver("< 0.6.4")`  must precede `semver("< 14.0.0")`.
Otherwise, `semver("< 0.6.4")` is never used.